### PR TITLE
Add design doc process

### PR DESCRIPTION
**Pull request by mishmosh**
Are the steps to share a document clear?

### CODE REVIEW

File: designdocs.md [hunk 1]
```diff
@@ -79,6 +80,19 @@ Obviously, we need specs too. But they often come later in the lifecycle, and ar
 
 We could capture a designdoc in a github issue, and this may in fact be a good convention. But a typical issue or bug has different content than what goes into a designdoc and sometimes you don’t want to parse all the discussion, you just want the conclusion.
 
+## Design Doc Process
+* Create new docs in the `Filecoin Community` > `Design Docs` folder.
+  * Check that you're using the GApps identity
```
Comment (ZenGround0) [inline designdocs.md]: I'm not 100% I know what this means.  Is there a separate `GApps` id I need to log into to do this?  Is this suggesting to sign in with your own appropriately chosen id?

File: designdocs.md [hunk 2]
```diff
@@ -79,6 +80,19 @@ Obviously, we need specs too. But they often come later in the lifecycle, and ar
...
+  * Use `DRAFT`, `IN REVIEW`, `ACCEPTED` to note status.
+* When you're ready, enable sharing for public comment
+  * Click "Share" button
+  * Click "Who has access" to open the advanced settings panel
+  * Choose "Anyone with the link can *comment*" from dropdown
```
Comment (ZenGround0) [inline designdocs.md]: nit: just now I ran into what seems to be a new google UI which required I toggled a slider to the right of the "Link Sharing" header before being able to hit the drop down menu.  Also now the drop down menu says "People at Filecoin with the link can comment".  The existing instructions were more than good enough, just noting the UI change.

File: designdocs.md (hunk 1 above)
Comment (anorth) [reply designdocs.md]: "If you have one, check that ..."

Comment (mishmosh) [reply designdocs.md]: Added “...identity of your choice”

File: designdocs.md (hunk 2 above)
Comment (mishmosh) [reply designdocs.md]: Thanks, I fixed these 2 details and also added “As of April 2019...” as a marker/insurance against future UI changes.

**Pull request merged**